iidm/iidm-converter-api/src/main/java/com/powsybl/iidm/AbstractOptions.java

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm;
+
+import java.util.HashSet;
+import java.util.Optional;
+import java.util.Set;
+
+/**
+ * @author Chamseddine BENHAMED <chamseddine.benhamed at rte-france.com>
+ */
+public abstract class AbstractOptions<T> {
+    protected Optional<Set<String>> extensions = Optional.empty();
+
+    protected IidmImportExportMode mode = IidmImportExportMode.UNIQUE_FILE;
```

Comment:
Is mode necessary in  AbstractOptions? Why not only in ExportOptions?

Reply:
Yes, I'm using mode in the ImportOptions too. it's a common attribute.

Reply:
I don't understand what could be the meaning of this option at import step because at import we do not control the splitting of data but just adapt to what has been done at export.